policyengine_us/parameters/gov/usda/snap/income/deductions/excess_medical_expense/standard.yaml

```diff
@@ -14,6 +14,12 @@ AZ:
 CA:
   2010-01-01: 0
   2021-10-01: 120
+  2024-10-01:
```

Comment:
did we also include the offset increase to the standard utility allowance?

Reply:
filed separate issue